eas.json

```
@@ -0,0 +1,18 @@
+{
+  "cli": {
+    "version": ">= 9.0.8"
+  },
```

Should we switch to remote version management?

https://docs.expo.dev/build-reference/app-versions/#remote-version-source

```suggestion
  "cli": {
    "appVersionSource": "remote"
  },
```

and maybe a `"autoIncrement": true` in the `production` object?

---

Of course we should! It will improve the DX a lot by avoiding to think about the version before pushing anything to the main branch, this is a killer feature to me (managing versions manually is a pitta)

Hopefully this won't conflict with existing versions, let's try it! Nice shot